src/main/java/uk/gov/digital/ho/hocs/CaseCreatorApplication.java

```diff
@@ -1,23 +1,13 @@
 package uk.gov.digital.ho.hocs;
 
-import lombok.extern.slf4j.Slf4j;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
-import org.springframework.retry.annotation.EnableRetry;
 
-import javax.annotation.PreDestroy;
-
-@Slf4j
 @SpringBootApplication
-@EnableRetry
 public class CaseCreatorApplication {
 
     public static void main(String[] args) {
         SpringApplication.run(CaseCreatorApplication.class, args);
     }
 
-    @PreDestroy
```

Comment:
Can we pop this back please, it is easier to track restarts this way.

Reply:
With the log-spam we have, I disagree but if you find it useful I'll add back in…

Reply:
I just find it easier to search for that line in kibana but you're right - if you can show me .